packages/hash/api/src/graphql/resolvers/knowledge/org/create-org.ts

```diff
@@ -0,0 +1,33 @@
+import { OrgModel } from "../../../../model";
+import {
+  MutationCreateOrgArgs,
+  ResolverFn,
+  Subgraph,
+} from "../../../apiTypes.gen";
+import { LoggedInGraphQLContext } from "../../../context";
+import { mapSubgraphToGql } from "../../ontology/model-mapping";
+
+export const createOrg: ResolverFn<
+  Promise<Subgraph>,
+  {},
+  LoggedInGraphQLContext,
+  MutationCreateOrgArgs
+> = async (
+  _,
+  { name, shortname, orgSize, linkResolveDepth, linkTargetEntityResolveDepth },
+  { dataSources: { graphApi }, userModel },
+) => {
+  const orgModel = await OrgModel.createOrg(graphApi, {
+    shortname,
+    name,
+    providedInfo: { orgSize },
+    actorId: userModel.entityId,
+  });
+
+  const orgRootedSubgraph = await orgModel.getRootedSubgraph(graphApi, {
+    linkResolveDepth: linkResolveDepth ?? 0,
+    linkTargetEntityResolveDepth: linkTargetEntityResolveDepth ?? 0,
```

Comment:
You don't need these if you define defaults in the schema and update the codegen config (see comment on the schema)
```suggestion
    linkResolveDepth,
    linkTargetEntityResolveDepth,
```

Reply:
Addressed in https://github.com/hashintel/hash/pull/1371/commits/a0f2c3312a5509000aec9bd036fc760f9051a651